src/reynir_correct/checker.py

```diff
@@ -354,7 +354,7 @@ def annotate(self, sent: Sentence) -> List[Annotation]:
                         start=0,
                         end=len(sent.tokens) - 1,
                         code="E001",
-                        text="Málsgreinin fellur ekki að reglum",
+                        text="Ekki tókst að þátta setningu, mögulega felst villa í henni",  # Formerly "Málsgreinin fellur ekki að reglum"
```

Comment:
Suggest using semicolon instead of comma ("setningu; mögulega")

Reply:
Done